src/App.jsx

```diff
@@ -1,8 +1,18 @@
+import { BrowserRouter, Routes } from "react-router-dom"
+import EmailList from "./components/EmailList"
+import Nav from "./components/Nav"
+
 function App() {
   return (
-    <div>
-      <h1 className="text-3xl font-bold underline text-red-500">Hello, Dragons</h1>
-    </div>
+    <BrowserRouter>
+      <div>
+          <Nav />
+          <EmailList /> 
```

Comment:
This EmailList should be on a route for / - this will enable you to make the inbox link in your menu link to /